docs/development/how-to/ide-setup/eclipse.md

```diff
@@ -53,5 +53,22 @@ Should look like this when configured: <br />
   - [EGit in Eclipse](http://www.eclipse.org/egit/) - with [tutorial
       ](http://www.vogella.com/tutorials/EclipseGit/article.html)
   - Run Configurations are checked in
-  - Run `./gradlew downloadAssets` one time to get images downloaded
-
+  - Run `./gradlew downloadAssets` (Windows: `gradlew.bat downloadAssets`) once in the main triplea folder to get images downloaded
+  - Run also the following commands (Windows: `gradlew.bat`) once in the main triplea folder if errors occur during gradle build for any of the tasks
+    - `./gradlew :spotlessApply`
```

Comment:
`./gradlew :spotlessApply` should recursively apply to all sub-projects. Did that not work? The enumeration of sub-projects should be unnecessary...